File: bubble_sort.py
array = [9,4,2,6,8,1,10]

def bubbleSort(array):
    index = 0 
    while index < len(array) -1:
        for count in range(index + 1, len(array)):
            if array[count] < array[index]:
                swap(count, index, array)
                index = 0
            else:
                index +=1 
    return array 


def swap(i, j, array):
	array[i], array[j] = array[j], array[i]


print(bubbleSort(array))

File: bubbleSort.js
array = [8,5,2,9,5,6,3]

bubbleSort = array => {
    let isSorted = false;
    while (!isSorted) {
        isSorted = true;
        for (let i = 0; i < array.length - 1; i++){
            if (array[i] > array[i+1]){
                swap(i, i+1, array);
                isSorted = false
            }
        }
    }
    return array;
}

swap = (i,j,array) => {
    const temp = array[j];
    array[j] = array[i];
    array[i] = temp;
}


console.log(bubbleSort(array))



